### ISSUE TITLE
Error while build: Can't resolve 'bdb/lib/DB'

### ISSUE THREAD

**Issue by suhailvs** [kyokan/bob-extension#5]
while build using command: `NODE_ENV=production npm run build`

i got error: 

```
ERROR in ./src/background/services/setting.ts
Module not found: Error: Can't resolve 'bdb/lib/DB' in '/home/suhail/github/handshake/bob-extension/src/background/services'
 @ ./src/background/services/setting.ts 5:11-32
 @ ./src/background/backgroundPage.ts
```

**Comment by suhailvs** [comment]
after a lot of try, replacing `'bdb/lib/DB'` by `'bdb/lib/db'` seems fixed this issue.

**Comment by rithvikvibhu** [comment]
Looks like fix in #1
